ui/js/component/formField/view.jsx

```diff
@@ -99,6 +117,16 @@ class FormField extends React.PureComponent {
     return this.refs.field.options;
   }
 
+  validate() {
+    if ("regexp" in this.props) {
+      if (!this.getValue().match(this.props.regexp)) {
+        this.showError("Invalid address.");
```

Comment:
This will not be detected on submit, only blur. The Right Way to do this is probably to have the form query each of its inputs about whether there's an error, but that's tricky to do with React/Redux for now. In this case the user will still find out because the form reports errors from the daemon.